tests/test.cpp

```diff
@@ -1566,24 +1571,24 @@ void FlexBuffersTest() {
 
   // Write the equivalent of:
   // { vec: [ -100, "Fred", 4.0 ], bar: [ 1, 2, 3 ], foo: 100 }
-  slb.Map([&]() {
-     slb.Vector("vec", [&]() {
-      slb += -100;  // Equivalent to slb.Add(-100) or slb.Int(-100);
-      slb += "Fred";
-      slb.IndirectFloat(4.0f);
-    });
+  slb.Map([](flexbuffers::Builder& slb2) {
```

Comment:
I still wanted to make sure the new code worked.  I guess I could add a version that works with old skool function pointers and new lambda / closures, WDYT?